pkg/validation/path.go

```diff
@@ -30,9 +30,7 @@ import (
 )
 
 var (
-	ErrInvalidJSONPath  error = fmt.Errorf("Invalid JSONPath")
-	ErrMismatchingTypes error = fmt.Errorf("Mismatching type(s)")
-	ErrWrongTypes       error = fmt.Errorf("Wrong type(s)")
+	ErrInvalidJSONPath  = fmt.Errorf("invalid JSONPath")
```

Comment:
I assume the rest of the errors were unused?

Reply:
Yes. For this PR I have just run "Inspect Code" in Goland and fixed some issues it found.